tests/cli/test_compose_sanity.sh

```diff
@@ -36,6 +36,11 @@ rlJournalStart
 
             rlRun -t -c "$CLI compose image $UUID"
             rlAssertExists "$UUID-root.tar.xz"
+
+            # because this path is listed in the documentation
+            rlAssertExists    "/var/lib/lorax/composer/results/$UUID/"
+            rlAssertExists    "/var/lib/lorax/composer/results/$UUID/$UUID-root.tar.xz"
+            rlAssertNotDiffer "/var/lib/lorax/composer/results/$UUID/$UUID-root.tar.xz" "$UUID-root.tar.xz"
```

Comment:
Inside results/UUID/ it is just root.tar.xz
The file is only named $UUID-root.tar.xz when retrieved with composer-cli compose image $UUID